src/com/puppetlabs/puppetdb/scf/migration_legacy.clj

```diff
@@ -0,0 +1,279 @@
+(ns com.puppetlabs.puppetdb.scf.migration-legacy
+  "Vestigial code required by various migrations.  Note that every
+  external call here increases the risk that future changes might
+  break a migration."
+  (:require
+   [com.puppetlabs.puppetdb.facts :as facts]
```

Comment:
@wkalt ok, I've sometimes just done that to provide more width in the future, and it lines up the imports and requires, but I have no strong feelings about it.